src/App.tsx

```diff
@@ -19,10 +40,25 @@ class App extends React.Component {
           >
             Learn React
           </a>
+          <p>Toggle this on and off to test out Redux:</p>
+          <p>
+            <button
+              onClick={this.handleToggleClick}
+              style={{ padding: '32px' }}
+            >
+              {toggleState === 'on' ? 'OFF' : 'ON'}
+            </button>
+          </p>
         </header>
       </div>
     );
   }
 }
 
-export default App;
+function mapStateToProps(state: ApplicationState) {
```

Comment:
I would be happy with doing this as well.

Reply:
For consistency, imo.

Reply:
that's the only reason?

Reply:
that's more than enough for me, plus `this` is correctly bound all the time, no need to think about that either.

Reply:
Alright, I'll change them to arrow functions. Consistency doesn't seem like the strongest rationale to me but if you both feel strongly, let's do it 🤷‍♀️  I will just need a lint rule otherwise I'll forget 😀 https://github.com/mozilla/addons-code-manager/issues/17#issuecomment-457371415